agent/src/agent/modules/logger.py

```diff
@@ -15,6 +15,7 @@ def get_logger(name, level=logging.DEBUG, stdout=False):
 
     if stdout:
         handler = logging.StreamHandler(sys.stdout)
+        handler.setLevel(logging.ERROR)
```

Comment:
setting global level error is unnecessary. it means that log will show only error messages, but info messages are also fine